src/elements/HiddenChipsBlock/hiddenChipsBlock.scss

```diff
@@ -9,7 +9,8 @@
   right: 0;
   z-index: 6;
   flex-direction: column;
```

Comment:
I know it's not you who wrote this `flex-direction: column;`, but it has no effect when there is no `display` with value `flex` or `inline-flex`.

I know this was not part of the requirement in the task, but I think that we can utilize more of the available space in this popover if we lay out the chips horizontally and wrap it.
So please apply these changes here:
```suggestion
  display: flex;
  flex-flow: row wrap;
```
to get this:
![image](https://user-images.githubusercontent.com/13918850/122280197-a1af6a00-cef1-11eb-9a51-292639220ce5.png)